src/views/RecIM/views/Home/index.js

```diff
@@ -184,7 +182,7 @@ const Home = () => {
               handleOpenWaiverForm(status);
             }}
             openWaiverForm={openWaiver}
-            setOpenWaiverForm={(bool) => setOpenWaiver(bool)}
+            setOpenWaiverForm={(waiverOpen) => setOpenWaiver(waiverOpen)}
```

Comment:
naming parameters after a type is not great practice

Reply:
I think it happened when I was mass renaming things, but I'll make sure I apply that to future instances!